multiversx_sdk_cli/cli_shared.py

```diff
@@ -177,10 +165,10 @@ def add_wallet_args(args: list[str], sub: Any):
         help="🔐 bool flag for signing transaction using ledger",
     )
     sub.add_argument(
-        "--ledger-address-index",
+        "--index",
```

Comment:
This is somehow ambiguous. Maybe `--sender-index`?

For _guardian_ & _relayer_ is more obvious what index means.

Reply:
renamed as follows:
```
--sender-wallet-index
--guardian-wallet-index
--relayer-wallet-index
```